.github/workflows/vast.yaml

```diff
@@ -920,6 +913,50 @@ jobs:
             done
           done
 
+  docker-compose:
+    needs:
+      - build-docker
+      - determine-version
+    if: github.event_name != 'workflow_dispatch'
+    name: Docker Compose
+    runs-on: ubuntu-20.04
+    env:
+      DOCKER_BUILDKIT: 1
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          submodules: recursive
+      - name: Install Docker Compose v2 CLI
+        uses: ndeloof/install-compose-action@v0.0.1
+        with:
+          version: latest
+          legacy: false
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Load VAST Docker Image
+        run: |
+          if [[ "$GITHUB_EVENT_NAME" == "pull_request" ]]; then
+            cache_to_ref="${{ needs.determine-version.outputs.head-ref-slug }}"
+          else
+            cache_to_ref="${{ github.sha }}"
+          fi
```

Comment:
this and the `buildx build` below are pretty verbose, would there be a way for the original build to complete the docker image so that we can just just `docker pull ghcr.io/tenzir/vast` ?